# HG changeset patch
# User mpoletiek <matthew.poletiek@gmail.com>
# Date 1606329087 21600
#      Wed Nov 25 12:31:27 2020 -0600
# Branch py3
# Node ID 055093bd90da079d927661375df69b1ac71346c6
# Parent  68534f20c1418ae8e4cc09f3ff468d0375ba843a
[mq]: py3syntaxfix.patch

diff --git a/chirp/drivers/vx6.py b/chirp/drivers/vx6.py
--- a/chirp/drivers/vx6.py
+++ b/chirp/drivers/vx6.py
@@ -871,5 +871,5 @@
                 elif setting == "password":
                     newval = self._encode_chars(newval, 4)
                 setattr(_settings, setting, newval)
-            except Exception, e:
+            except (Exception, e):
                 raise
diff --git a/chirp/ui/mainapp.py b/chirp/ui/mainapp.py
--- a/chirp/ui/mainapp.py
+++ b/chirp/ui/mainapp.py
@@ -1137,7 +1137,7 @@
 
         query = "http://chirp.danplanet.com/query/rb/1.0/app_direct" \
                 "?loc=%s&band=%s&dist=%s" % (loc, band, dist)
-        print query
+        print(query)
 
         # Do this in case the import process is going to take a while
         # to make sure we process events leading up to this
